sdk/cosmos/azure-cosmos-spark_3-1_2-12/src/main/scala/org/apache/spark/source/MySource.scala

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package org.apache.spark.source
```

Comment:
I assume this file should not be checked-in yet?

Reply:
thanks for catching this. Yes some local work. It shouldn't get checked in.

I removed it.